cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -1097,9 +1105,27 @@ IOResult<void> export_input_data_county_timeseries(
 
                     extrapolated_rki[county][day]((size_t)InfectionState::SusceptibleImprovedImmunity +
                                                   age_group_offset) =
-                        std::min(S + S_pv + S_v, std::max(0.0, double(extrapolated_rki[county][day](
-                                                                   (size_t)InfectionState::SusceptibleImprovedImmunity +
-                                                                   age_group_offset))));
+                        std::min(
+                            S_v -
+                                extrapolated_rki[county][day]((size_t)InfectionState::ExposedImprovedImmunity +
+                                                              age_group_offset) -
+                                extrapolated_rki[county][day](
+                                    (size_t)InfectionState::InfectedNoSymptomsImprovedImmunity + age_group_offset) -
+                                extrapolated_rki[county][day](
+                                    (size_t)InfectionState::InfectedNoSymptomsImprovedImmunityConfirmed +
+                                    age_group_offset) -
+                                extrapolated_rki[county][day]((size_t)InfectionState::InfectedSymptomsImprovedImmunity +
+                                                              age_group_offset) -
+                                extrapolated_rki[county][day](
+                                    (size_t)InfectionState::InfectedSymptomsImprovedImmunityConfirmed +
+                                    age_group_offset) -
+                                extrapolated_rki[county][day]((size_t)InfectionState::InfectedSevereImprovedImmunity +
+                                                              age_group_offset) -
+                                extrapolated_rki[county][day]((size_t)InfectionState::InfectedCriticalImprovedImmunity +
+                                                              age_group_offset),
+                            std::max(0.0,
+                                     double(extrapolated_rki[county][day](
+                                         (size_t)InfectionState::SusceptibleImprovedImmunity + age_group_offset))));
```

Comment:
I cannot comment below but is S_PI not lacking the Dead_PI change from above?

Reply:
In the context of extrapolation, only DeadNaive is currently used. All reported dead are written to this compartment.
 See
https://github.com/DLR-SC/memilio/blob/2de2900be8f493c57715f5636e9ad5f64ddcff43/cpp/models/ode_secirvvs/parameters_io.h#L1197
Above theres also an short explanation. 
For completeness, we could use the Dead_PI here as well, but that could lead to an additional source of errors (incorrectly initialized values there) and communicate a misunderstanding to the user that they think the variable is being used.

Reply:
The problem is: You now introduced the Death_PI/II compartments to the read_input_data which should give exactly the same result as the extrapolation. This is now not consistent anymore.

Reply:
its now consistent. Although, we definitly have to reduce the doubling of code in the extrapolation and initialization. Therefore, i have openend issue #794.